content/docs/functions/json_build_object.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Postgres json_build_object function
+subtitle: "Builds a JSON object out of a variadic argument list."
+enableTableOfContents: true
+---
+
+
+Use `json_build_object` to construct a JSON object from a set of key-value pairs, creating a JSON representation of a row or set of rows.
```

Comment:
Maybe add some context or example, something like:
"This can increase performance compared to converting query results to JSON on the application side."